src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -292,11 +293,19 @@ export class EditDataEditor extends BaseEditor {
 			this._setSashDimension();
 
 			this._register(this._sash.onPositionChange(position => this._doLayout()));
+			this._register(this.themeService.onDidColorThemeChange(e => this.updateTheme(e)));
 		}
 
 		this._sash.show();
 	}
 
+	//update sash theme
+	private updateTheme(theme: IColorTheme): void {
+		if (this._resultsEditorContainer && this._resultsEditorContainer.style.borderTopColor !== '') {
```

Comment:
I think the borderTopColor check here is unnecessary - it makes sense to always set it regardless